Fix warning "File 'F:\workspace_work\1\s\artifacts\obj\RedistSharedFx.Layout\Release\win-x64\shared\Microsoft.AspNetCore.App\5.0.0-preview.6.20275.3.version' was not added to the package."

Part of https://github.com/dotnet/aspnetcore/issues/22240

Happens during the SiteExtension build. Full text below:

> F:\workspace\_work\1\s\.dotnet\sdk\5.0.100-preview.5.20258.4\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(198,5): warning NU5119: File 'F:\workspace\_work\1\s\artifacts\obj\RedistSharedFx.Layout\Release\win-x64\shared\Microsoft.AspNetCore.App\5.0.0-preview.6.20275.3\.version' was not added to the package. Files and folders starting with '.' or ending with '.nupkg' are excluded by default. To include this file, use -NoDefaultExcludes from the commandline [F:\workspace\_work\1\s\src\SiteExtensions\Runtime\Microsoft.AspNetCore.Runtime.SiteExtension.pkgproj]

> F:\workspace\_work\1\s\.dotnet\sdk\5.0.100-preview.5.20258.4\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(198,5): warning NU5119: File 'F:\workspace\_work\1\s\artifacts\obj\RedistSharedFx.Layout\Release\win-x64\shared\Microsoft.NETCore.App\5.0.0-preview.6.20271.10\.version' was not added to the package. Files and folders starting with '.' or ending with '.nupkg' are excluded by default. To include this file, use -NoDefaultExcludes from the commandline [F:\workspace\_work\1\s\src\SiteExtensions\Runtime\Microsoft.AspNetCore.Runtime.SiteExtension.pkgproj]

--- a/src/SiteExtensions/Runtime/Microsoft.AspNetCore.Runtime.SiteExtension.pkgproj
+++ b/src/SiteExtensions/Runtime/Microsoft.AspNetCore.Runtime.SiteExtension.pkgproj
@@ -38,7 +38,7 @@
 
   <Target Name="ResolveReferenceItemsForPackage" DependsOnTargets="ResolveReferences" BeforeTargets="_GetPackageFiles">
     <ItemGroup>
-      <Content Include="$(DotNetUnpackFolder)\**\*.*" Condition="$(DotNetAssetRootUrl) != ''" PackagePath="content\%(RecursiveDir)" />
+      <Content Include="$(DotNetUnpackFolder)\**\*.*" Exclude="$(DotNetUnpackFolder)\**\.*" Condition="$(DotNetAssetRootUrl) != ''" PackagePath="content\%(RecursiveDir)" />
       <Content Include="%(NativeContent.Identity)" PackagePath="content\ancm\%(NativeContent.HandlerPath)" />
     </ItemGroup>
   </Target>

